Dev-Corner/slides/visualizations.html

```diff
@@ -62,112 +62,233 @@
-
-                            - list
-                            - list
-                            - list
+                            ## Why visualizations?
+                            
+                            <img src="../images/vis_alignment_plaintext.png" class="no-border" width="70%"/>
                         </script>
                     </section>
 
                     <section data-markdown>
                         <script type="text/template">
-                            ### Subpart1.2
-
-                            With an image...
-
-                            <img src="../../shared/images/GTNLogo1000.png" alt="GTNLogo1000" class="no-border" width=30%/>
+                            ## Why visualizations?
+                            
+                            <img src="../images/vis_alignment.png" class="no-border" width="99%"/>
                         </script>
                     </section>
                 </section>
 
                 <section>
                     <section data-markdown>
                         <script type="text/template">
-                            ### <i class="fa fa-question-circle" aria-hidden="true"></i> Questions
+                            ### <i class="fa fa-exclamation-circle" aria-hidden="true"></i> Types of visualization in Galaxy
 
-                            - Major question that would be addressed in this tutorial (same as the markdown)
-                            - Second question
+                            - As Galaxy tool (wrapper)
+                            - As 3rd party app (./display_applications)
+                            - Trackster
+                            - Interactive environment
+                            - As visualization <i> plugin</i>
 
                             <aside class="notes">
-                                Oh hey, these are some notes. They'll be hidden in your presentation, but you can see them if you open the speaker notes window.
-
-                                To see the notes, hit **'s'** on your keyboard
+                                Examples:
+                                 - as tool: iReport, jBrowse
+                                 - as 3rd party app: IGV, UCSC
                             </aside>
                         </script>
                     </section>
 
                     <section data-markdown>
                         <script type="text/template">
-                            ### <i class="fa fa-bullseye" aria-hidden="true"></i> Objectives
-
-                            - First objective of this tutorial (same as the markdown)
-                            - Second objective
-                            - Third objective
-                            - ...
+                            ### Pro's and con's
+
+                            <table>
+                                <tr><td>Type</td>        <td>i_active</td>   <td>Workflow</td>   <td>Shed</td>  <td>tweakable<td>  <td>serv_idx<td></tr>
```

Comment:
It's become clearest with the aside but ca you explicite a little the headers or maybe use a png?